src/main/java/org/asciidoctor/maven/AsciidoctorMojo.java

```diff
@@ -242,10 +319,9 @@ private void setDestinationPaths(OptionsBuilder optionsBuilder, final File sourc
 
     protected Asciidoctor getAsciidoctorInstance(String gemPath) throws MojoExecutionException {
         Asciidoctor asciidoctor = null;
-        if (gemPath == null) {
-             asciidoctor = Asciidoctor.Factory.create();
-        }
-        else {
+        if (gemPath == null || gemPath.isEmpty()) {
```

Comment:
If I remember correctly, there is a different. The empty string is the default, and that does need to be passed to create() for Asciidoctor 1.5.x. You are correct that 1.6.0 fixes this. What I don't remember is how to make the value null. I think if you put an empty tag (`<gemPath/>`), that makes the value null. A null value means "don't isolate" in this context.